### ISSUE TITLE
@shared/public-api is missing all the enums / interfaces for queries

### ISSUE THREAD

**Issue by skewty** [thingsboard/thingsboard-extensions#55]
If I don't use my own ../query, which has copies of these, the widgets will fail to load and browser console shows that they cannot be found.

Example:
```ts
import {
  BooleanOperation,
  EntityData,
  EntityKeyType,
  EntityKeyValueType,
  EntityDataQuery,
  FilterPredicateType,
  NumericOperation,
  StringOperation,
} from "../query"
```

All of the above enums / interfaces are missing from `@shared/public-api`.

Using IDE suggestion / feature to auto-import missing, it does:

```ts
import {
  EntityDataQuery,
  BooleanOperation,
  EntityData,
  EntityKeyType,
  EntityKeyValueType,
  FilterPredicateType,
  NumericOperation,
  StringOperation,
} from "@shared/models/query/query.models"
```

The worst part is that `clean install -P yarn-start` completes without error.

These are certainly part of the public api because these are all offered in the "ThingsBoard REST API" page at `/swagger-ui/#`.

Can these please get added into the public-api?

**Comment by skewty** [comment]
I see some similar type cleanup slated for version 3.6.2 inclusion. 
Reference: https://github.com/thingsboard/thingsboard/pull/9010

Will this issue be fixed for 3.6.2 as well?

**Comment by vvlladd28** [comment]
This problem will be fixed in the version 3.6.2
